source/sync/error-handling/client-resets.txt

```diff
@@ -82,121 +84,219 @@ with the client's history. The most common causes of client resets are:
 Handle a Client Reset
 ---------------------
 
-The SDKs automatically detect the need for client resets. You
-can handle client resets in your application using one of the available
-**client reset strategies**:
+The SDKs automatically detect the need for client resets. They can 
+automatically perform a client reset in most cases, except in the 
+event of a breaking schema change.
 
-.. note:: Breaking Changes Require Manual Recovery
+During an automatic client reset, the client:
 
-   You can only handle a client reset triggered by breaking changes
-   using the :ref:`manually recover unsynced changes
-   <manually-recover-unsynced-changes>` strategy.
+1. Downloads a fresh copy of the realm from the backend.
+#. Performs a diff to figure out the set of steps required to
+   bring the original (local) copy of the realm to the same state as
+   the fresh copy of the backend.
+#. Applies that set of steps to transform the local realm into a state
+   where it can sync with the backend. 
+   
+   If there are no schema changes, the SDK attempts to recover all local 
+   changes that have not yet synced to the backend. It also applies 
+   any inserts, updates, and deletes from the backend that haven't yet 
```

Comment:
```suggestion
   any inserts, updates, and deletes from the backend that hasn't yet 
```

Reply:
"Haven't" in this instance refers to "inserts" "updates" and "deletes", so the plural is the correct form to use, grammatically-speaking.